<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplaySettings</class>
 <widget class="QWidget" name="DisplaySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Right Lane</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QCheckBox" name="ckRLOWallSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>L Upper Outer Wall</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QCheckBox" name="ckRWallWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="ckRWallSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QCheckBox" name="ckLShoulderWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QCheckBox" name="ckRoofWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="ckAllSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Right Wall</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QCheckBox" name="ckLUOWallWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QCheckBox" name="ckRLOWallWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>R Lower Outer Wall</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Surface</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QCheckBox" name="ckOWallFloorWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Wireframe</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>R Upper Outer Wall</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="ckAllWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="ckRoofSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="ckLWallSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="ckLShoulderSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="ckRShoulderSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QCheckBox" name="ckRUOWallSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="ckLLaneWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QCheckBox" name="ckRUOWallWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Right Shoulder</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="1">
    <widget class="QCheckBox" name="ckOWallFloorSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QCheckBox" name="ckRShoulderWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Environment Floor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QCheckBox" name="ckEnvirFloorWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>All</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="ckRLaneWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>L Lower Outer Wall</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_16">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Roof</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Left Wall</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="ckRLaneSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="ckLLaneSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Left Lane</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QCheckBox" name="ckEnvirFloorSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Outer Wall Floor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QCheckBox" name="ckLLOWallWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="ckLLOWallSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Left Shoulder</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="ckLUOWallSurface">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QCheckBox" name="ckLWallWireframe">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QCheckBox" name="ckHighlightSelection">
     <property name="text">
      <string>Highlight Selected Chunks</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
